### PR TITLE
Better loggin

### PR DESCRIPTION
**Pull request by 1day2die**
Added somewhat more detailed activity log with clickable Name of Causer

image

Issue https://github.com/ControlPanel-gg/dashboard/issues/186

### CODE REVIEW

**Comment by 1day2die** [comment]
somehow this includes the changes made to the user profiles showing the Discord Card...im confused by Github

**Comment by anondev-sudo** [comment]
You need to create new branch and commit there

**Comment by 1day2die** [comment]
yeah i cant untill these pull-requests are merged

File: resources/views/profile/index.blade.php [hunk 1]
```diff
@@ -213,6 +213,23 @@ class="fa fa-coins mx-1"></i>{{$credits_reward_after_verify_discord}}
                                                             <p>You are verified!</p>
                                                         </div>
                                                     </div>
+                                                    <div class="row">
```
Comment (AVMG20) [inline resources/views/profile/index.blade.php:216]: this doesn't seem to have anything to do with better logs? for clarity make this a separate pull next time :)

**Pull request merged**